content/updates/2021-03/4-Users-roles-and-groups-import-export.md

```diff
@@ -0,0 +1,22 @@
+---
+pim_announcement_alt_img: "A user import job"
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Import and export your users, roles and groups
+::: meta-data type="New" features="Productivity" available="late March" in="EE,GE" link-to-doc="../articles/manage-your-users-via-import-export.html"
+:::
+
+You now have at your disposal several new jobs that will allow you to import and export your users, their roles as well as their groups. These new jobs are very handy in case of mass edition of your users.
```

Comment:
```suggestion
We introduced new job profiles to allow you to import and export your users including their roles and groups. These new jobs are very handy in case of a mass edition of your users.
```

Reply:
We avoid past in news, as agreed with @camillebourjade

Reply:
fixed in the suggestion

Reply:
Thanks!
I feel like the word "including" induces that whenever you export users, you will also export their roles. Here, it's not what I want to say. There will be export (and import) profiles for the 3 entities: user, role and group. You know what I mean?

Reply:
I also put back "introduced" because having it in present mode does not make sense - I'm checking with Camille, and will edit it if needed afterwards.